openff/interchange/components/interchange.py

```diff
@@ -109,7 +109,9 @@ def from_smirnoff(
             positions are taken from the molecules in topology, if present on all molecules.
         charge_from_molecules : `List[openff.toolkit.molecule.Molecule]`, optional
             If specified, partial charges will be taken from the given molecules
-            instead of being determined by the force field.
+            instead of being determined by the force field. All molecules in this list
+            must have partial charges assigned and must not be isomorphic with any other
+            molecules in the list.
```

Comment:
(I had to grab this change manually because ... GitHub detected this involved a deleted line, and for some reason that precludes multi-line suggestions from being applied. Boo!)